src/view/payment-mobile.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { InputField } from '../component/field';
+import { Header, Title } from '../component/header';
+import { CancelButton, Button, GlasButton } from '../component/button';
+import Card from '../component/card';
+import LightningBoltIcon from '../asset/icon/lightning-bolt';
+import { FormStretcher, FormText, FormSubText } from '../component/form';
+import QRCodeScanner from '../component/qrcode-scanner';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  btnWrapper: {
+    alignSelf: 'stretch',
+    flexDirection: 'row',
+  },
+  btnLeft: {
+    flex: 1,
+    borderRightWidth: 0.5,
+    borderRightColor: color.purple,
+  },
+  btnRight: {
+    flex: 1,
+    borderLeftWidth: 0.5,
+    borderLeftColor: color.purple,
+  },
+});
+
+const PaymentView = ({ store, nav, payment }) => (
+  <Background color={color.purple}>
+    <Header color={color.purple}>
+      <Button disabled onPress={() => {}} />
+      <Title title="Lightning Payment">
+        <LightningBoltIcon height={12} width={6.1} />
+      </Title>
+      <CancelButton onPress={() => nav.goHome()} />
+    </Header>
+    {store.payment.useScanner ? (
+      <QRCodeScanner
+        onQRCodeScanned={({ data }) => payment.readQRCode({ data })}
+      />
+    ) : (
+      <PaymentCard store={store} payment={payment} />
+    )}
+    <View style={styles.btnWrapper}>
+      <GlasButton onPress={() => payment.pasteAddress()} style={styles.btnLeft}>
```

Comment:
I can't paste the "normal" way (i.e. long press on the input field causing a paste option to pop up) on the SE. However, I can on the 8+. /shrug

Reply:
Really? It worked for me.